.github/workflows/common.yml

```diff
@@ -153,8 +153,9 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Run Merge Gatekeeper
-        uses: percona-platform/merge-gatekeeper@v1.0.2
+        uses: upsidr/merge-gatekeeper@v1
```

Comment:
@atymchuk  Please add this action to allow-list. Also, do we want to use major versions like `@v1`, or exact like `@v1.1.0`?

Reply:
It depends. For a tool like this with only 20-ish starts on github, I would apply zero trust. Therefore, I'd suggest we pin it to the exact version.